src/MongoDB.Driver.Core/Core/Misc/SemanticVersion.cs

```diff
@@ -285,7 +342,17 @@ public static bool TryParse(string value, out SemanticVersion result)
         /// </returns>
         public static bool operator >=(SemanticVersion a, SemanticVersion b)
         {
-            return !(a < b);
+            if (object.ReferenceEquals(a, null))
+            {
+                if (object.ReferenceEquals(b, null))
+                {
+                    return true;
+                }
+
+                return false;
+            }
+
+            return a.CompareTo(b) >= 0;
```

Comment:
This change is not needed. Original implementation is correct:

    return !(a < b);

Reply:
This change is not needed. Original implementation is correct.

There is no point in changing something that is correct.

It is also about reusing the logic that already exists in the `<` implementation.

Reply:
Done.